### PR TITLE
kh360-react-621-create-table-component-refactor

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Create Table Component
[#621](https://github.com/nerubia/kh360-react/issues/621)

### CODE REVIEW

**Review by emmaacha — APPROVED** [review]

**Review by emmaacha — CHANGES_REQUESTED** [review]
Please add cursor-pointer style when user hovers on a row (if row is clickable)

**Comment by chadiegilnerubia** [comment]
> Please add cursor-pointer style when user hovers on a row (if row is clickable)

okey done :3

File: src/components/ui/table/table.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                {column}
+              </th>
+            ))}
+          </tr>
+        </thead>
+        <tbody>
+          {data.map((item, index) => (
+            <tr
+              key={index}
+              className='bg-white border-b hover:bg-slate-100 text-black hover:cursor-pointer'
```
Comment (emmaacha) [inline src/components/ui/table/table.tsx:37]: Not all rows in the existing tables are clickable, maybe we should set a flag to check if it's clickable or not.

Comment (emmaacha) [reply src/components/ui/table/table.tsx:37]: Also, remove `hover:` for `cursor-pointer`

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/components/ui/table/table.tsx (hunk 1 above)
Comment (emmaacha) [reply src/components/ui/table/table.tsx]: > Also, remove `hover:` for `cursor-pointer`

Up on this

**Review by emmaacha — CHANGES_REQUESTED** [review]
> Also, remove `hover:` for `cursor-pointer`

Up 👆

**Comment by chadiegilnerubia** [comment]
> > Also, remove `hover:` for `cursor-pointer`
> 
> Up 👆

Donee :3

**Review by gborgonia-ir — APPROVED** [review]

**Comment by emmaacha** [comment]
> > > Also, remove `hover:` for `cursor-pointer`
> > 
> > 
> > Up 👆
> 
> Donee :3

`hover:` is still there

File: src/components/ui/table/table.tsx (hunk 1 above)
Comment (chadiegilnerubia) [reply src/components/ui/table/table.tsx]: Oppss, i have removed it now

**Review by emmaacha — APPROVED** [review]

**Pull request merged**